app/views/tag/_location.html.erb

```diff
@@ -15,7 +15,7 @@
       <% end %>
       if( $(this).attr("submit_to") ) {
         modalUrl += firstVar ? '?' : '&';
-        modalUrl += 'url=' + encodeURIComponent(locationInput.attr("submit_to"));
+        modalUrl += 'url=' + encodeURIComponent($(this).attr("submit_to"));
```

Comment:
@gauravahlawat81 I didn't quite understand what you are doing here. Why did you use `$(this)` ?

Reply:
I don't know why, but this doesn't seem to work, when we select using `locationInput` it returns undefined. So I though if we are searching if `submit_to` exists or not using `$(this)`, we can also use the value in same way,

Please let me know if you have any better ideas.

Reply:
I don't have any better ideas per se...but I wasn't too sure how this was working.
Anyway thanks for explaining.
Cheers!